docs-ref-conceptual/includes/cli-install-linux-dnf.md

```diff
@@ -8,7 +8,7 @@ ms.custom: devx-track-azurecli, linux-related-content
 
 ## Before you begin
 
-- Azure CLI [2.38.2](/cli/azure/release-notes-azure-cli#june-21-2024) (released on June 21, 2024) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft isn't providing other updates or bug fixes for these distributions.
+- Azure CLI [2.38.2](/cli/azure/release-notes-azure-cli#june-21-2024) (released on June 21, 2024) is the last version supported on RHEL 7 and Centos 7 and will receive no additional security fixes. Microsoft also isn't providing updates or bug fixes for these distributions.
```

Comment:
Please also change this in `docs-ref-conceptual/includes/cli-install-linux-zypper.md`

Reply:
@bebound, sorry, I'm not following.  cli-install-linux-zypper.md doesn't mention RHEL 7 or Centos 7.

Reply:
> Azure CLI [2.38.2](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#june-21-2024) (released on June 21, 2024) is the latest version supported on SUSE Linux Enterprise Server (SLES) and OpenSUSE (Leap) and will continue to receive security fixes until the end of June 2024. Microsoft isn't providing other updates or bug fixes for these distributions.